src/app/iconography/icon-selection.ts

```diff
@@ -5,13 +5,13 @@
  */
 import { Component } from "@angular/core";
 import { CoreShapes } from "../../clarity-icons/shapes/core-shapes";
-import { BasicShapes } from "../../clarity-icons/shapes/basic-shapes";
+import { EssentialShapes } from "../../clarity-icons/shapes/essential-shapes";
```

Comment:
Oh yes. Thanks for catching this. I completely forgot to keep the old basic-shapes modules.